samples/build/build_buildpacks-v3_cr.yaml

```diff
@@ -1,15 +1,16 @@
+---
 apiVersion: build.dev/v1alpha1
 kind: Build
 metadata:
-  name: buildpacks
+  name: buildpack-nodejs-build
 spec:
   source:
     url: https://github.com/sclorg/nodejs-ex
   strategy:
     name: buildpacks-v3
-    namespace: otaviof
+    namespace: openshift
```

Comment:
I wouldn't create the build CRs in the `openshift` namespace because it is a namespace which devs typically don't have access to.

However, since,
-  "admins" create build strategies, 
- "developers" use those to start builds

.. we should have BuildStrategies in the `openshift` namespace and Builds in some random user namespace `foo`.

Reply:
@sbose78 Note that in the snippet it's referring to `spec.strategy.namespace`, therefore `openshift` is the namespace where the `BuildStrategy` resource is located.

Reply:
I do agree with the reasoning, by the way. And, for this operator we should define a default namespace, for instance `openshift-buildstrategies`, in order to store the strategies shipped with the operator. And then, in the examples we can always employ it.

Reply:
> @sbose78 Note that in the snippet it's referring to spec.strategy.namespace, therefore openshift is the namespace where the BuildStrategy resource is located.

Right right, my bad!

Reply:
For the sake of our examples, I moved everything to `build-examples` so that our yamls work out of the box :)

https://github.com/redhat-developer/build#try-it 

In future, BuildStrategy would just become cluster-scoped, therefore namespace specification would not be needed.

-